src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -355,33 +355,32 @@ protected AnalyzerDriver(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerM
             }
         }
 
-        private async Task ProcessEventAsync(CompilationEvent e, CancellationToken cancellationToken)
+        private static readonly Task s_emptyTask = Task.FromResult<object>(null);
+
+        private Task ProcessEventAsync(CompilationEvent e, CancellationToken cancellationToken)
```

Comment:
As I understand it, the Async suffix means "task returning" whereas the 'async' (contextual) keyword is a (language) artifact of the implementation.